sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractForm.java

```diff
@@ -400,6 +401,11 @@ protected boolean isConfiguredServer(String countryCode) {
 		return FacadeProvider.getConfigFacade().isConfiguredCountry(countryCode);
 	}
 
+	protected CountryReferenceDto getServerCountry() {
```

Comment:
@MateStrysewske which would be the best class for this method, since we have a similar sequence of code in LocationsField?